packages/addon-dev/sample-rollup.config.js

```diff
@@ -1,15 +1,28 @@
-import babel from '@rollup/plugin-babel';
+import { babel } from '@rollup/plugin-babel';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+
 import { Addon } from '@embroider/addon-dev/rollup';
 
+import packageJson from './package.json';
+
 const addon = new Addon({
   srcDir: 'src',
   destDir: 'dist',
 });
 
+const extensions = ['.js', '.ts'];
+
 export default {
+  // This tells rollup to follow the node-resolution algorithm for finding
+  // dependencies / and files within those dependencies
+  nodeResolve({ extensions }),
```

Comment:
In general, I don't think we want folks to do this. This will make rollup embed your dependencies (not allow them to be resolved at application build time), which would be quite unfortunate. It would mean that the addon author in question would have to redeploy a new patch release for any security/patch releases in dependencies... 

More details below in the `externals` section.